source/includes/_tipser-elements.md

```diff
@@ -193,3 +193,43 @@ import { ShopComponent } from '@tipser/tipser-elements';
 
 Note: the shop component is updating the top-level page URL (when it's tabs are clicked). For this reason, please double check if it won't interfere with your web framework. 
 For the same reason, it's not recommended to include more than one `<ShopComponent />` on a single page. 
+
+
+## Style Guidelines ##
+
+Tipser Elements are the "building blocks" designed to fit your page as much as possible. We created the styling in a way that delivers a nice feel & looks from the start, but also allows you to change them easily to fit your unique sense of style. 
+
+### Collection Element ###
+
+```css
+.te-product-card {
+    font-size: 14px;
+}
+```
+
+The collection consists of the products shown in Product Cards. The font-family used in the description section is inherited from your website's styles, and the font-size is expressed in the relative `em` units controlled in `.te-product-card` class. The default value used there is `12px`, which you can easily change by adding to your CSS following style:
+
+All the description elements (product name, brand and price) will become bigger / smaller according to the value you specify in the `px` unit. If you wish to change single description element, please use its specific class names: 
+
+`.te-product-card-name` <br/>
+`.te-product-card-brand`<br/>
+`.te-product-card-price`<br/>
+and:<br/>
+`.te-product-card-sales-price`<br/>
+`.te-product-card-price-regular-price`<br/>
+for product with campaign rules.
```

Comment:
this is very nice.

Just a note for later: What would be even better, is to allow user to see examples with changes instantly. That could be quite possible with Live Documentation such as Storybook or Styleguidist.